woocommerce/woocommerce-ios
# Login: UI updates for the native Jetpack setup flow

Part of #8075 


### Description

This PR update UI for the screens in the native Jetpack setup flow following the latest updates [p1669336075073439-slack-C046HDZL87J]. Changes include:
- Error screens: Changed copies and added site address information.
- Site credentials login screen: Updated the text field style, updated the title, and removed the disclaimer about approval.
- Jetpack setup screen: 
  - Display the "Error" text underneath any step that fails
  - Removed the “Approval required” warning from the connection step.
  - Changed the “Authorising” step name – in the step list and the web view.
  - Changed the “Connection approved” status to just “Connected”.
- Empty store picker: updated assets for both light and dark modes.

### Testing instructions

Prerequisite: make sure that you have a test site without Jetpack.

- Log out of the app or skip login onboarding if needed.
- On the prologue screen, select "Log In" or "Continue with WordPress.com" based on the A/B test variant you get.
- Log in with your WordPress.com account.
- If your account is not associated with any site, notice that the empty store picker has the correct image for both dark and light modes.
- On the store picker, select Add a Store > Connect an existing site.
- Enter the address of your test store and tap Continue.
- On the Jetpack error screen, notice that the UI is updated correctly. Select Install Jetpack or Connect Jetpack.
- When the site credential login screen is presented, notice the UI is updated correctly. 
- Enter the correct credentials for a shop manager or administrator account.
- On the Jetpack setup screen, when the setup reaches the connection step, notice that a web view is presented for approving the Jetpack connection. The UI and copies for both screen should be correct.
- Tap Cancel to dismiss the web view, notice that a new view is presented regarding the interrupted Jetpack connection with the updated design.


### Screenshots

Empty store picker:
| Light mode | Dark mode |
| ----- | ----- |
| <img src="https://user-images.githubusercontent.com/5533851/203929462-6f8a27f2-eae0-40a5-a00d-a287dea49b2a.png" width=320 /> | <img src="https://user-images.githubusercontent.com/5533851/203929532-25b3795b-da13-4571-83bc-64c9845196eb.png" width=320 /> |

Error screens:
| Jetpack setup | Jetpack connection | Setup interrupted |
| ----- | ----- | ----- |
| <img src="https://user-images.githubusercontent.com/5533851/203930639-77c150ca-9d84-4d32-b82a-c6e2e5391c33.png" width=320 /> | <img src="https://user-images.githubusercontent.com/5533851/203930721-32a63fd2-32d9-4984-abc8-b2bf6746e802.png" width=320 /> | <img src="https://user-images.githubusercontent.com/5533851/203930737-5a56575a-727d-40a4-a2ec-33b0b85befb3.png" width=320 /> |

Site credential screen:

<img src="https://user-images.githubusercontent.com/5533851/203933656-af9680b6-cff7-4a80-9fda-82ccdd04f88b.png" width=320 />

Setup flow:

https://user-images.githubusercontent.com/5533851/203934054-f43c1191-2c88-40af-8eb4-8631e1623723.mp4


---
